assets/scripts/dialogs/__tests__/GeotagDialog.test.js

```diff
@@ -54,43 +70,66 @@ describe('GeotagDialog', () => {
     expect(wrapper.queryByText('Confirm location')).not.toBeInTheDocument()
   })
 
-  // TODO: Fix these tests
-  it.skip('allows a location to be confirmed when the current signed-in user is the street owner', () => {
+  /*
+  the tests for location pop up already test whether the popup shows up or not
+  tests for StreetMetaGeotag also cover some of this behavior
+  do we have a setup available for mocking a signed-in vs anonymous user? */
```

Comment:
To answer this question, usually we look at the [`user.signInData`](https://github.com/streetmix/streetmix/blob/master/assets/scripts/store/slices/user.js#L4) property in Redux to know if this user is signed in or not. We can likely add it to `initialState` since the `user` object is there.